classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php

```diff
@@ -217,6 +217,11 @@ function importPeerReviews($args, $request) {
 						$body .= PKPString::stripUnsafeHtml($comment->getComments());
 					}
 				}
+
+				// Add reviewer recommendation
+				$recommendation = $reviewAssignment->getLocalizedRecommendation();
+				$body .= __('reviewer.article.recommendation') . ': ' . $recommendation . "<br>\n";
```

Comment:
This assumes a right-to-left presentation, which might not be appropriate in some languages. I'd suggest adding a new locale key for this.

Reply:
Done. https://github.com/pkp/pkp-lib/pull/3096/commits/e6e4209ccd5c6467d2419ca3dc348ee4a9bede24

I came across `submission.recommendation` locale key which I have reused.